src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf

```diff
@@ -428,19 +428,19 @@
             </trans-unit>
             <trans-unit id="110">
                 <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
-                <target state="needs-review-translation">پسوند فایل نامعتبر است ({{ extension }}). پسوندهای مجاز {{ extensions }} هستند.</target>
+                <target>پسوند فایل ({{ extension }}) نامعتبر است. پسوندهای مجاز {{ extensions }} هستند.</target>
             </trans-unit>
             <trans-unit id="111">
                 <source>The detected character encoding is invalid ({{ detected }}). Allowed encodings are {{ encodings }}.</source>
-                <target state="needs-review-translation">رمزگذاری کاراکتر تشخیص داده شده نامعتبر است ({{ detected }}). رمزگذاری‌های مجاز {{ encodings }} هستند.</target>
+                <target>رمزگذاری کاراکتر تشخیص داده شده ({{ detected }}) نامعتبر است. رمزگذاری‌های مجاز {{ encodings }} هستند.</target>
             </trans-unit>
             <trans-unit id="112">
                 <source>This value is not a valid MAC address.</source>
-                <target state="needs-review-translation">این مقدار یک آدرس MAC معتبر نیست.</target>
+                <target>این مقدار یک آدرس MAC معتبر نیست.</target>
             </trans-unit>
             <trans-unit id="113">
                 <source>This URL is missing a top-level domain.</source>
-                <target state="needs-review-translation">این URL فاقد دامنه سطح بالا است.</target>
+                <target>این آدرس دارای دامنه نمی‌باشد.</target>
```

Comment:
? (from GPT)
```suggestion
                <target>این URL فاقد دامنه سطح بالا است.</target>
```

Reply:
(basically GPT prefers its own previous version :) )

Reply:
GPT translates this sentence word by word.
In Persian, we do not use `دامنه سطح بالا`, instead we use `دامنه`.
I translate `URL` to its relevant word in Persian, So it's better to use `آدرس` instead of `URL`.

So, As a native speaker of Persian, it's correct and there is no need to change.

> (basically GPT prefers its own previous version :) )